ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -1562,6 +1562,31 @@ def verify_consumer_storagecluster(sc_data):
     assert re.match("\\d+[PT]", granted_capacity)
 
 
+def verify_managedocs_security():
+    """
+    Check ocs-osd-deployer-operator permissions:
+    1. Verify `runAsUser` is not 0
+    2. Verify `SecurityContext.allowPrivilegeEscalation` is set to false
+    3. Verify `SecurityContext.capabilities.drop` only has capabilities
+    KILL, MKNOD, SETGID, SETUID
```

Comment:
@ebondare I don't see the verification implementation for this step

Reply:
Yes, I needed some additional information on dropped capabilities. Fixed